docs/source/conf.py

```diff
@@ -33,6 +33,7 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
+    'tools.sphinxext.c7n_schema',
```

Comment:
I thought you asked me to put this in tools so it didn't count against coverage?  I needed to put `__init__.py` into tools so that the extension could be imported.